src/entity/object/FallingBlock.php

```diff
@@ -97,13 +104,27 @@ protected function entityBaseTick(int $tickDiff = 1) : bool{
 		if($this->closed){
 			return false;
 		}
+		$this->fallTime++;
 
 		$hasUpdate = parent::entityBaseTick($tickDiff);
 
 		if(!$this->isFlaggedForDespawn()){
 			$world = $this->getWorld();
 			$pos = $this->location->add(-$this->size->getWidth() / 2, $this->size->getHeight(), -$this->size->getWidth() / 2)->floor();
 
+			if($this->getBlock()->getId() === VanillaBlocks::ANVIL()->getId()) {
+
+				$collidedEntities = $world->getCollidingEntities($this->getBoundingBox());
+				foreach($collidedEntities as $ent)
```

Comment:
Braces must be used.

Reply:
This variable is also redundant. The above statement could be moved directly into the foreach.